utils/appliance.py

```diff
@@ -39,6 +39,25 @@
     from utils.hosts import setup_providers_hosts_credentials
 
 
+class logger_wrap(object):
```

Comment:
Well, it might be used in another module which would possibly could have different keyword name. But I am happy if we keep it for appliance for now. I see taht if we want to make it universal, it can be done with backward compatible changes.